internal/cmd/keys.go

```diff
@@ -159,6 +160,15 @@ func newKeysListCmd() *cobra.Command {
 				}
 			}
 
+			machineNames := make(map[uid.ID]string)
+			for _, key := range keys {
+				machineRes, err := client.GetIdentity(key.IssuedFor)
```

Comment:
Identities names are not unique? That seems like a problem because most of our CLI commands seem to accept names. Doesn't that put us in a position where if someone accidentally creates a duplicate name, then they are no longer able to remove it or do anything with it from the CLI?  Should we have a unique constraint on the identity name?

Reply:
Ohh, maybe I misunderstood. This is list access keys. So there may be multiple access keys per machine, is that what you mean?